checkov/common/util/docs_generator.py

```diff
@@ -73,7 +73,7 @@ def add_from_repository(registry, checked_type: str, iac: str):
         add_from_repository(arm_parameter_registry, "parameter", "arm")
     if framework == "secrets" or framework == "all":
         for check_id, check_type in CHECK_ID_TO_SECRET_TYPE.items():
-            printable_checks_list.append((check_id, check_type, "secrets", check_type, check_type, "secrets"))
+            printable_checks_list.append((check_id, check_type, "secrets", check_type, "secrets"))
```

Comment:
@rotemavni can you share a screenshot of the md? I'm not sure missing a column would not make the table look funky

Reply:
We spoke